src/tools/run-make-support/src/external_deps/rustc.rs

```diff
@@ -300,6 +301,30 @@ impl Rustc {
         self
     }
 
+    /// Configuration for the child process’s standard input (stdin) handle.
+    ///
+    /// See [`std::process::Command::stdin`].
+    pub fn stdin<T: Into<Stdio>>(&mut self, cfg: T) -> &mut Self {
+        self.cmd.stdin(cfg);
+        self
+    }
```

Comment:
Suggestion: actually, can you move these impls to the `impl_common_helpers!` macro? These should be command-specific, not rustc specific.